### ISSUE TITLE
Create accessibility audit template

### ISSUE THREAD

**Issue by andrewhayward** [WordPress/gutenberg#50931]
## What problem does this address?

#50166 requires that an accessibility audit be run across the @wordpress/components package. There are currently over 100 components, so this would very much need to be a community effort to keep timescales reasonable.

## What is your proposed solution?

In order to encourage better community participation, [an issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository) should be created to guide contributors through the process of auditing a component.

This should not aim too high in terms of accessibility experience, but should reasonably assume contributors know how to use (for example) browser extensions, developer tools, etc.

**Comment by ciampo** [comment]
This would be a great way to have a standard way to perform accessibility audits.

I imagine the tricky part is that not all folks may have access to a full suite of testing tools, especially the ones that require different OSs. We should therefore aim to create a comprehensive audit guide without being excessive in the amount of work required, or we incur the risk of alienating most folks. Maybe it could be a good idea if we split the "must do" from the "nice to have" tasks.

Finally, let's not forget that we all have a finite amount of time, and therefore we've got to be wise in choosing what to work on — let's focus, therefore, on the most impactful ways of auditing, and on fixing the most important issues.

**Comment by joedolson** [comment]
The purpose of auditing is to identify issues, and should be handled completely separately from fixing them. 

I think that the audit template should allow for different people to perform different tasks, rather than assuming any one person will completely audit a component. But there are some critical aspects of testing (like verifying that keyboard behavior is functional when a screen reader is running) that require tools that not everybody has access to or familiarity with.

Issues need to contain task lists that can be checked off as each test is completed, but the tests can be performed by a variety of participants.
